session4/src/com/itschool/homework/Challenge5.java

```diff
@@ -0,0 +1,36 @@
+package com.itschool.homework;
+
+import java.util.Scanner;
+public class Challenge5 {
+
+    public static void main(String[] args) {
+//
+//        int x = 10;
+//        int y = 11;
+//        //assignment and compound assignment operators
+//
+//        System.out.println(x*=5);
+//        System.out.println(x%=3);
+//        System.out.println(y%=3);
+//        System.out.println(x+=11);
+//        System.out.println(x-=1);
+//        System.out.println(x/=2);
+
+        Scanner input = new Scanner(System.in);
```

Comment:
this `input` object is unused and can be removed if you don't to insert the variables from keyboard (you are hard-coding them below).

Reply:
this is still unused. You can remove it if you don't need it.